cypress/integration/aaaaaaaaaaaaaaaaaaaaaaaa/add_a_product_to_a_pricelist_schema_and_create_a_new_PLV.js

```diff
@@ -0,0 +1,184 @@
+/*
+ * #%L
+ * metasfresh-e2e
+ * %%
+ * Copyright (C) 2019 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+import { humanReadableNow } from '../../support/utils/utils';
+import { PriceListSchema, PriceListSchemaLine } from '../../support/utils/price_list_schema';
+import { Builder } from '../../support/utils/builder';
+import { applyFilters, selectNotFrequentFilterWidget, toggleNotFrequentFilters } from '../../support/functions';
+import { ProductPrices } from '../../page_objects/product_prices';
+import { PriceList, PriceListVersion } from '../../support/utils/pricelist';
+
+let date = humanReadableNow();
```

Comment:
Why `let` and not `const` ?

Reply:
coz when testing i'm modifying the date to some literal. and it's easier to have `let date` and on the next line i change the value (only 1 row to comment) vs `const date` and have to ping-pong between them :D